src/Tizen.Multimedia.MediaPlayer/Player/SphericalVideo.cs

```diff
@@ -0,0 +1,329 @@
+                ThrowIfFailed(Player, "Failed to get the level of the zoom");
+
+            return value;
+        }
+
+        /// <summary>
+        /// Gets the level of the zoom when it is spherical .
+        /// </summary>
+        /// <remarks>Remark.</remarks>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="InvalidOperationException">The player is not in the valid state.</exception>
+        /// <seealso cref="GetZoom()"/>
+        /// <since_tizen> 5 </since_tizen>
+        public void SetZoom(float level)
+        {
+            Player.ValidateNotDisposed();
+
+            if (level < 1.0F || 10.0F < level)
+            {
+                throw new ArgumentOutOfRangeException(nameof(level), level, "Valid level is 1.0 to 10.0");
+            }
+
+            NativePlayer.SetZoom(Player.Handle, level).ThrowIfFailed(Player, "Failed to set the level of the zoom.");
+        }
+
+        /// <summary>
+        /// Gets or sets the spherical mode.
+        /// </summary>
+        /// <remarks>The player must be in the <see cref="PlayerState.Ready"/>, <see cref="PlayerState.Playing"/>,
+        /// or <see cref="PlayerState.Paused"/> state.</remarks>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="InvalidOperationException">The player is not in the valid state.</exception>
+        /// <since_tizen> 5 </since_tizen>
+        public bool IsSphericalMode
+        {
+            get
+            {
+                NativePlayer.IsSphericalMode(Player.Handle, out var value).
+                    ThrowIfFailed(Player, "Failed to get the spherical mode of the player");
+                return value;
+            }
+
+            set
+            {
+                Player.ValidateNotDisposed();
+                NativePlayer.SetSphericalMode(Player.Handle, value).
+                    ThrowIfFailed(Player, "Failed to set the spherical mode of the player");
+            }
+        }
+
+
+        /// <summary>
+        /// Gets the properties of the audio.
+        /// </summary>
+        /// <returns>A <see cref="DirectionOfView"/> that contains the audio stream information.</returns>
+        /// <remarks>
+        /// Remark
+        /// </remarks>
+        /// <exception cref="ObjectDisposedException">
+        /// The <see cref="Multimedia.Player"/> that this instance belongs to has been disposed of.
+        /// </exception>
+        /// <exception cref="InvalidOperationException">
+        /// The <see cref="Multimedia.Player"/> that this instance belongs to is not in the valid state.
+        /// </exception>
+        /// <since_tizen> 5 </since_tizen>
+        public DirectionOfView GetDirectionOfView()
+        {
+            Player.ValidateNotDisposed();
+
+            NativePlayer.GetDirectionOfView(Player.Handle, out var yaw,
```

Comment:
How about merging below 2 lines (252~253) to one.

Reply:
ok